scattr/workflow/rules/mrtpipelines/preproc.smk

```diff
@@ -1,42 +1,29 @@
-if dwi_dir:
-    print(f"Searching {dwi_dir} for dwi and mask images...")
-
-
 rule nii2mif:
     input:
-        dwi=(bids_dwi(suffix="dwi.nii.gz") if dwi_dir else inputs["dwi"].path),
-        bval=(
-            bids_dwi(suffix="dwi.bval")
-            if dwi_dir
-            else re.sub(".nii.gz", ".bval", inputs["dwi"].path)
-        ),
-        bvec=(
-            bids_dwi(suffix="dwi.bvec")
-            if dwi_dir
-            else re.sub(".nii.gz", ".bvec", inputs["dwi"].path)
-        ),
-        mask=(
-            bids_dwi(suffix="mask.nii.gz") if dwi_dir else inputs["mask"].path
-        ),
+        dwi=inputs_dwi["dwi"].path,
+        bval=re.sub(".nii.gz", ".bval", inputs_dwi["dwi"].path),
```

Comment:
Not a new issue, but do the `.`s here need to be escaped because this is a regex?

Reply:
Not in this case - I think yes if the string had been `r".nii.gz"`, but I haven't had an issue with this so far.